packages/token-introspection/README.md

```diff
@@ -0,0 +1,43 @@
+# Token Introspection
+
+This package supports [GNAP](https://datatracker.ietf.org/doc/html/draft-ietf-gnap-core-protocol) [token introspection](https://datatracker.ietf.org/doc/html/draft-ietf-gnap-resource-servers#section-3.3) requests from a **resource server** to an **authorization server**.
+
+It includes:
+
+- Exported TypeScript types generated from a GNAP token introspection OpenAPI specification
+- A NodeJS client for sending token introspection requests
```

Comment:
Do we want to mention here that the token introspection specifications extend from the Open Payments AS spec at all?